frontend/src/Resources/index.jsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  View, Image, Text,
+  StyleSheet,
+} from 'react-native';
+import factoryIcon from '../../assets/buildings/factory.png';
+import mineIcon from '../../assets/buildings/mine.png';
+import cookieIcon from '../../assets/troop/cookie.png';
+import goldIcon from '../../assets/gold.png';
+import Colors from '../common/colors';
+import { CardView } from '../common/components';
+
+const styles = StyleSheet.create({
```

Comment:
please fix the styling so the resource items are aligned vertically evenly somehow because right now it looks like this

![Screen Shot 2019-12-17 at 10 21 01](https://user-images.githubusercontent.com/30794349/70959518-7b6bfc80-20b7-11ea-9b75-83db075c21a6.png)

instead of something similar like this:

![Screen Shot 2019-12-17 at 10 26 51](https://user-images.githubusercontent.com/30794349/70959653-ca199680-20b7-11ea-81ac-6096c8414449.png)